src/lingrex/trimming.py

```diff
@@ -138,14 +138,16 @@ def trimmed(self,
         for idxss in candidates:
             if not isinstance(idxss, list):
                 idxss = [idxss]
-            trimmed = False
+            trimmed = []
             for idx in idxss:
                 current_skeleton = [c for i, c in skeleton if i not in idxs | {idx}]
                 if any(subsequence_of(s, current_skeleton) for s in skeletons):
                     # Trimming this site leaves a "big enough" remainder.
                     idxs.add(idx)
-                    trimmed = True
-            if not trimmed:
+                    trimmed.append(True)
+                else:
+                    trimmed.append(False)
+            if not all(trimmed):
```

Comment:
@LinguList @Tarotis and here's the fix for the fix: We now quit trimming if not **all** sites with the same gap ratio have been trimmed - which is how I understand the idea of the algorithm. Otherwise it could happen that a site with lower gap ratio than one of the remaining sites is trimmed.